### DISCUSSION TITLE
Filter expressions - string match?

### DISCUSSION THREAD

**Discussion by fgreco55** [milvus-io/milvus#28132] [Q&A and General discussion]
I'm doing a query to find if a substring is in one of my collections.  I don't see an operator that allows me to find string/substring/regexp matches for a specific attribute of a collection, eg, "mycolumn has ".*date$".   Any assistance appreciated.

I'm writing tests to confirm my docs are loaded into Milvus before I do a vector retrieval.

**Comment by xiaofan-luan** [comment]
currently milvus doesn't support post prefix or wildcard match. the like operator only support prefix matching.

We may deliver more filtering functions on milvus 2.4 but remember the performance might still be bad.
